include/MainWindow.h

```diff
@@ -81,9 +81,20 @@ class MainWindow : public QMainWindow
 	///
 	bool mayChangeProject(bool stopPlayback);
 
-	void autoSaveTimerStart()
+	// Auto save timer intervals
+	const static int m_autoSaveLongSeconds = 120;
+	const static int m_autoSaveShortSeconds = 10;
+	const static int m_autoSaveLongTime = m_autoSaveLongSeconds * 1000;
+	const static int m_autoSaveShortTime = m_autoSaveShortSeconds * 1000;
+
+	void autoSaveTimerReset( int msec = m_autoSaveLongTime )
+	{
+		m_autoSaveTimer.start( msec );
+	}
+
+	int getAutoSaveTimerInterval()
 	{
-		m_autoSaveTimer.start( 1000 * 60 );  // 1 minute
+		return m_autoSaveTimer.interval() / 1000;
```

Comment:
What is the point of constantly dividing by 1000?  I can understand if the GUI needs to do this, but it seems like unnecessary within the code.